src/services/messengerSync/testConsumer.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+var amqp = require('amqplib/callback_api');
+
+amqp.connect('amqp://192.168.99.100', function(error0, connection) {
+    if (error0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)